features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Base/Errors/AuthorizedError.jsx

```diff
@@ -0,0 +1,124 @@
+                                    <Typography variant='subtitle1' gutterBottom>
+                                        {message.body}
+                                        <span style={{ color: 'green' }}>
+                                            {' '}
+                                            {response ? response.statusText : ''}
+                                            {' '}
+                                        </span>
+                                    </Typography>
+                                    {message.more}
+                                    <Box py={5}>
+                                        <Box pb={2}>
+                                            <Typography variant='subtitle1'>
+                                                <FormattedMessage
+                                                    id='Base.Errors.ResourceNotFound.more.links'
+                                                    defaultMessage='You may check the links below'
+                                                />
+                                            </Typography>
+                                        </Box>
+                                        <Link to='/apis/' style={{ marginRight: 8 }}>
+                                            <Button variant='contained' color='primary'>
+                                                <FormattedMessage
+                                                    id='Base.Errors.ResourceNotFound.api.list'
+                                                    defaultMessage='API List'
+                                                />
+                                            </Button>
+                                        </Link>
+                                        <Link to='/api-products/'>
+                                            <Button variant='contained' color='primary'>
+                                                <FormattedMessage
+                                                    id='Base.Errors.ResourceNotFound.api.product.list'
+                                                    defaultMessage='API Product List'
+                                                />
+                                            </Button>
+                                        </Link>
+                                    </Box>
+                                </Grid>
+                                <Grid item xs={12} md={6}>
+                                    {/* Image */}
+                                    <Image404 style={{ fontSize: 400, fill: '#ccc' }} />
+                                </Grid>
+                            </Grid>
+                        </Box>
+                    </Paper>
+                </Box>
+            </Container>
+        </>
+    );
+};
+
+AuthorizedError.defaultProps = {
+    message: {
+        title: <FormattedMessage
+            id='Base.Errors.AuthorizedError.default_tittle'
+            defaultMessage='Authorized Error'
```

Comment:
`Authorized Error` doesn't give a proper meaning right ?

Shall we say Error 403: Forbidden or Resource Forbidden kind of message?